src/components/prescription-expanded.component.tsx

```diff
@@ -49,16 +54,16 @@ const PrescriptionExpanded: React.FC<{
     <div className={styles.expandedTabsParentContainer}>
       <div className={styles.expandedTabsContainer}>
         <Tabs>
-          <TabList>
+          <TabList aria-label="Tab List">
             {tabs.map((tab: TabItem, index: number) => (
               <Tab key={index} className={styles.orderTabs}>
                 {tab.name}
               </Tab>
             ))}
           </TabList>
           <TabPanels>
-            {tabs.map((tab: TabItem) => (
-              <TabPanel>{tab.component}</TabPanel>
+            {tabs.map((tab: TabItem, index) => (
+              <TabPanel key={index}>{tab.component}</TabPanel>
```

Comment:
I know you're only supposed to use keys as a "last resort", but admittedly I don't really understand the reasoning, I added it here just to suppress the warning message...